_static/js/custom.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    window.onload = function () {
+        var isApiPresent = document.getElementsByTagName('redoc').length;
+        if(isApiPresent) {
+            window.addEventListener("hashchange", function(){
+                var locationHash = location.hash.split('#')[1],
+                    elementToScroll = null;
+                if(locationHash.split('/')[0] !== 'section'&& locationHash.split('/')[0] !== 'tag' ) {
+                    elementToScroll = document.querySelector('[href="'+ location.hash +'"]');
+                }
+                if(elementToScroll){
```

Comment:
I don't see a point to this `if` statement. Can't you scroll as soon as `elementToScroll` is set (in previous `if` statement)?

Reply:
This is more like POC, it will be fixed in another ticket